_08-ex.Rmd

```diff
@@ -50,7 +50,7 @@ plot(germany_borders)
 st_write(germany_borders, "germany_borders.gpkg")
 ```
 
-E5. Download the global monthly minimum temperature with a spatial resolution of five minutes using the **geodata** package.
+E5. Download the global monthly minimum temperature with a spatial resolution of 5 minutes using the **geodata** package.
```

Comment:
Why this? Not consistent with previous change from 8 to eight, right?

Reply:
This is the case I mentioned to you last week in Prague. I have zero idea why...

Reply:
:shrug:

Reply:
No big deal either way... "5" is shorter so would advocate for that throughout. Not pedantic enough to care, that much...